src/lib/components/aeris-catalog-cart/aeris-catalog-cart.vue

```diff
@@ -45,48 +46,48 @@
 
       <ul class="item-list">
         <template id="cartTemplate" v-for="item in this.cartContent">
-				<li class="cart-line">
-					<div>
-						<aeris-metadata-international-field class="cart-collection-name" :content="item.collectionName" :lang="lang" no-label-float></aeris-metadata-international-field>
-						<span>&nbsp;- {{item.fileNumber}}
-							<i class="fileIcon fa fa-file-o"></i>
-							<span class="files-size">({{computeFileSize(item.fileSize)}})</span>
-						</span>
-					</div>
-					<div>
-					</div>
-					<i class="fa fa-trash" @click="removeCartItem(item.collectionId)"></i>
-				</li>
-				<div class="filter-description" v-if='item.filterDescription' >
-				{{item.filterDescription}}
-				</div>
-				</template>
+          <li class="cart-line" :key="item.id">
+            <i class="fa fa-times" @click="removeCartItem(item.collectionId)"></i>

```

Comment:
un petit tooltip sur la croix? par exemple: remove ou remove this selection from download lists etc

Reply:
Bonne idée! Je rajouterai ça!